docs/3.0.0-beta.x/guides/external-data.md

```diff
@@ -0,0 +1,96 @@
+# Fetching external data
+
+This guide will explain how to fetch data in an external service and use it in your app.
+
+In this example we will see how to daily fetch Docker pull count and store it in database.
+
+## Content Type settings
+
+First we will have to create a Content Type. Let say the Content Type name will be `hit` and will have a `date` and a `count` attributes.
```

Comment:
First, we need to create a Content Type so let's call it `hit`; it will have a `date` and a `count` attributes.

Reply:
This sentence just seems odd to me 🤔 

Maybe something like: First we need to create a Content Type, in this example we will call it `hit` with a `date` and `count` attribute.

Reply:
I agree!